src/betterproto/__init__.py

```diff
@@ -25,13 +25,16 @@
     get_type_hints,
 )
 
+import dataclasses
+
```

Comment:
Why was dataclasses moved down to here?

Reply:
I believe it's because of py36 compat dataclass is considered as third party.

Reply:
switched the python version to 37 for now to resolve that.